charts/prometheus-rabbitmq-exporter/values.yaml

```
@@ -63,6 +63,7 @@ rabbitmq:
 ## Additional labels to set in the Deployment object. Together with standard labels from
 ## the chart
 additionalLabels: {}
+deploymentAnnotations: {}
```

I would like to keep the naming consistent:

```suggestion
additionalAnnotations: {}
```